### PR TITLE
add github authentication link for stripe pay button for  recurring p…

### PR DESCRIPTION
**Pull request by michal-macioszczyk**
…ayment

### CODE REVIEW

File: kuma/static/js/payments-handler.js [hunk 1]
```diff
@@ -242,6 +245,12 @@
             value: selectedAmount * 100
         });
 
+        if (requestUserLogin) {
+            requestUserLogin.classList.remove('hidden');
+            form.addClass('hidden');
```
Comment (matthewhall) [inline kuma/static/js/payments-handler.js]: I think we should be consistent with the DOM APIs we're using. You're using the `classList` API above and then jQuery here. Can we pick one and stick to it?

If we've decided that we'd like to start slowly moving over to use more native DOM APIs then that's cool. You can use them with jQuery elements like this:

```
form.get(0).classList.add('hidden');
```

File: kuma/static/js/payments-handler.js [hunk 2]
```diff
@@ -450,7 +459,19 @@
         }
     }
 
+    function redirectUserToLogin(event){
+        event.preventDefault();
+        var gitHubLink = $(this).attr('href');
+        var gitHubNext = $(this).data('next');
+        var getFormFields = form.serialize();
+        gitHubLink = gitHubLink + '&next='+ gitHubNext + '?' + encodeURIComponent(getFormFields);
```
Comment (matthewhall) [inline kuma/static/js/payments-handler.js]: `gitHubLink += '&next=' + gitHubNext + '?' + encodeURIComponent(getFormFields);` will do the same here and it a little more concise.

File: kuma/static/js/payments-handler.js [hunk 3]
```diff
@@ -450,7 +459,19 @@
         }
     }
 
+    function redirectUserToLogin(event){
```
Comment (matthewhall) [inline kuma/static/js/payments-handler.js]: Add a space before `{`.

Comment (matthewhall) [inline kuma/static/js/payments-handler.js]: JSDoc comment for this please.

**Comment by joshJarr** [comment]
Nice one @matthewhall, addressed comments. Let me know if there's anything else.

**Pull request closed without merging**